packages/kbn-es-archiver/README.mdx

```diff
@@ -14,4 +15,31 @@ This tool also has a CLI which can be used to save indexes to new archives or lo
 
 To teach the ES Archiver how to talk to ES and Kibana it is ideal to start ES and Kibana using `node scripts/functional_test_servers --config some/config/file.ts` and then use the same `--config` flag when running `node scripts/es_archiver` so that it can access the location and authorization information about the ES instace from the FTR config file.
 
-Additional information about what functionality the CLI provides can be found by running `node scripts/es_archiver --help`
\ No newline at end of file
+Additional information about what functionality the CLI provides can be found by running `node scripts/es_archiver --help`
+
+## Performance Option Parameter
+
+We now have a performance parameter for the es-archiver#load(), entry-point 
+function.
+This parameter is optional, with defaults:
+
+ - Batch size: 5000
+ - Concurrency (maximum number of bulk requests that can be active in parallel): 4 
+
+### Usage in Test Files
+
+To control the batch size and concurrency
+
+#### Example 
+
+```typescript
+await esArchiver.load('x-pack/test/functional/es_archives/getting_started/shakespeare', {
+  performance: {
+    batchSize: 5000,
+    concurrency: 4,
+  },
+});
+```
+According to our benchmarks, in a servless environment, this archive went from
+loading in around 2 and half minutes, to under 18 seconds.
+This was with a batch size of 5000 and concurrency of 4, as shown above.
```

Comment:
I'd suggest 2 things:
1. Move this section up to where we describe the default (after the `Concurrency` bullet point and before the `### Usage ...` heading)
2. Reword it slightly so it's target the default values in general, not only for one specific archive. E.g. something like
    ```
    According to our benchmarks, these default settings are giving the best results in terms of loading time for the majority of the archives. However, there might be cases where different settings are needed, so they can be overridden when loading an archive.
    ```